src/XMLimport.h

```diff
@@ -50,7 +50,7 @@ class XMLimport : public QXmlStreamReader
 public:
     XMLimport(Host*);
 
-    bool importPackage(QIODevice* device, QString packageName = QString(), int moduleFlag = 0);
+    bool importPackage(QIODevice*, QString packageName = QString(), int moduleFlag = 0, QString* pVersionString = Q_NULLPTR);
```

Comment:
I don't see it being recommended as the Qt way anywhere - it's more like a Qt workaround of anyone < C++11. Nevermind me though, I was wrong on the tools account and since Q_NULLPTR has made its way into the codebase might as well stick with it.